fhir-server-test/src/test/java/com/ibm/watson/health/fhir/server/test/ServerSpecTest.java

```diff
@@ -522,7 +522,7 @@ public void testSearchInvalidResourceType() {
         WebTarget target = getWebTarget();
         Response response = target.path("NotAResourceType").queryParam("notasearchparameter", "foo").request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
         assertResponse(response, Response.Status.BAD_REQUEST.getStatusCode());
-        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class), "The virtual resource types feature is not enabled for this server");
+        assertExceptionOperationOutcome(response.readEntity(OperationOutcome.class), "The resource type 'NotAResourceType' is not allowed.");
```

Comment:
I assume its right.  Excited to get our integration tests running as part of PRs again so its easier to verify.